back/src/Services/GaugeManager.ts

```diff
@@ -0,0 +1,41 @@
+import {Gauge} from "prom-client";
+import {clientEventsEmitter} from "./ClientEventsEmitter";
+
+class GaugeManager {
+    private nbClientsGauge: Gauge<string>;
+    private nbClientsPerRoomGauge: Gauge<string>;
+    private nbGroupsPerRoomGauge: Gauge<string>;
+
+    constructor() {
+        this.nbClientsGauge = new Gauge({
+            name: 'workadventure_nb_sockets',
+            help: 'Number of connected sockets',
+            labelNames: [ ]
+        });
+        this.nbClientsPerRoomGauge = new Gauge({
+            name: 'workadventure_nb_clients_per_room',
+            help: 'Number of clients per room',
+            labelNames: [ 'room' ]
+        });
+        this.nbGroupsPerRoomGauge = new Gauge({
+            name: 'workadventure_nb_groups_per_room',
+            help: 'Number of groups per room',
+            labelNames: [ 'room' ]
+        });
```

Comment:
This gauge will allow us to know how many groups there are at a given time in a given room.
It's cool.

But we should also add another gauge that will keep track of the total number of groups ever started in a room.
It is the exact same gauge as the previous one... except we don't decrease it when a group is deleted.
This is useful for event organizers, as a measure of total engagement of the users.